### PR TITLE
[MM-844]: Added server testcase for store/welcome_store.go

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
### Summary
Added server testcase for store/welcome_store.go

### CODE REVIEW

File: calendar/store/welcome_store_test.go [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+			},
+			assertions: func(t *testing.T, resp string, err error) {
+				require.Equal(t, resp, "")
+				require.EqualError(t, err, "failed plugin KVGet: KVGet failed")
+			},
+		},
+		{
+			name: "Success loading user welcome post",
+			setup: func(mockAPI *MockPluginAPI) {
+				mockAPI.ExpectedCalls = nil
```
Comment (raghavaggarwal2308) [inline calendar/store/welcome_store_test.go]: we should do it in the loop

File: calendar/store/welcome_store_test.go [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+}
+
+func (m *MockPluginAPI) KVGet(key string) ([]byte, *model.AppError) {
+	args := m.Called(key)
+	data, _ := args.Get(0).([]byte)
+	if err := args.Get(1); err != nil {
+		return nil, err.(*model.AppError)
+	}
+	return data, nil
+}
```
Comment (raghavaggarwal2308) [inline calendar/store/welcome_store_test.go]: We have created this in another testcases file as well. Can we make it generalised

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

File: calendar/store/welcome_store_test.go (hunk 2 above)
Comment (Kshitij-Katiyar) [reply calendar/store/welcome_store_test.go]: can do but we will need to change the base of this PR to `MM-843`

Comment (raghavaggarwal2308) [reply calendar/store/welcome_store_test.go]: I think we should do it

**Review by raghavaggarwal2308 — APPROVED** [review]

**Comment by Kshitij-Katiyar** [comment]
Created a PR with all these changes on MM remote hence closing this one

**Pull request closed without merging**